resolwe_bio/tools/parse_diffexp.py

```diff
@@ -20,6 +20,7 @@ def parse_arguments():
     parser.add_argument('--logodds', help="Log Odds column name", type=str)
     parser.add_argument('--logfc', help="logfc column name", type=str)
     parser.add_argument('--stat', help="Statistics column name", type=str)
+    parser.add_argument('--output', help='Output JSON file')
```

Comment:
The parameter should have a default value, because the script crashes not if `--output` not given. I suggest the default is `de_data.json`.